### PR TITLE
Nerfs Tactical Cock Flattening

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

A smaller dick = more aerodynamic to hunt valids.

Except no longer.

## Why It's Good For The Game

I hate ERP code

## Changelog
:cl:
fix: Adds a check for enlargement when your junk goes below the enlargement values.
/:cl:

### CODE REVIEW

**Review by silicons — APPROVED** [review]

File: modular_citadel/code/modules/arousal/organs/breasts.dm [hunk 1]
```diff
@@ -89,6 +89,8 @@
 		QDEL_IN(src, 1)
 		return
 	var/enlargement = FALSE
+	if(enlargement && rounded_cached < 16)
+		enlargement = FALSE
```
Comment (Ghommie) [inline modular_citadel/code/modules/arousal/organs/breasts.dm:93]: Unreachable code, `enlargement` was already FALSE.

File: modular_citadel/code/modules/arousal/organs/penis.dm [hunk 2]
```diff
@@ -40,6 +40,8 @@
 	var/rounded_length = round(length)
 	var/new_size
 	var/enlargement = FALSE
+	if(enlargement && rounded_length < 21)
+		enlargement = FALSE
```
Comment (Ghommie) [inline modular_citadel/code/modules/arousal/organs/penis.dm:44]: Idem.

**Review by Ghommie — CHANGES_REQUESTED** [review]
I wonder what you are trying to accomplish here...

**Comment by necromanceranne** [comment]
Well apparently this isn't going to do anything so...probably best I just close this? Whatever issues are arising from movement speed as a result of genital size is a problem I more or less don't care enough to fix and would rather just remove entirely and hardcap sizes. But whatever...

**Pull request closed without merging**